### DISCUSSION TITLE
Support for Nook Simple Touch (NST)

### DISCUSSION THREAD

**Discussion by broomerr** [plateaukao/einkbro#53] [Q&A]
Hello, @plateaukao.
Any chance that EinkBro will support Nook Simple Touch?
I see that existed releases require Android 5+ so it could  not be  used on NST stright away.

Nook Simple Touch is Android 2.1
Processor: Texas Instruments OMAP 3621
arch: arm7+neon

https://en.wikipedia.org/wiki/Nook_Simple_Touch

**Comment by plateaukao** [comment]
Hi 

Android 2.1 was released in 2009, 12 years ago. I believe that most implementations in EinkBro are not compatible with Android 2.1.  :)
